res/css/views/right_panel/_ThreadPanel.scss

```diff
@@ -131,24 +168,20 @@ limitations under the License.
     }
 
     .mx_AutoHideScrollbar {
-        border-radius: 8px;
-    }
```

Comment:
What do you think about this solution? https://github.com/matrix-org/matrix-react-sdk/pull/7105
I also fiddled around with this, when I wanted to get the hidden scrollbar with rounded corners working with the [`TimelineCard`](https://github.com/matrix-org/matrix-react-sdk/pull/7112) (Chat card in the right panel). I wanted it to be based on the same solution as the threads so I played around with the threads as well.
```css
.mx_AutoHideScrollbar {
          border-radius: 8px;
          padding-right: 0px;
          -ms-overflow-style: none;  // IE and Edge
          scrollbar-width: none;  // Firefox
          
          &::-webkit-scrollbar {
              display: none; // Chrome, Safari, Opera
          }
}
```
Hiding the scrollbar with the browser and not do the margin/padding trick gave good results with rounded corners.

Reply:
I don't think that hiding the scrollbar is an option. They are an important indicator to orientate yourself in the app, and  I believe they serve some accessibility purpose too